site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -120,9 +132,11 @@ export const WorkspaceReadyPage = ({
           ),
         }}
         isUpdating={workspaceState.matches("ready.build.requestingUpdate")}
+        isRestarting={isRestarting}
```

Comment:
@BrunoQuaresma one of the workspace props is `failedBuildLogs`. Looks like this is completely handled by a hook that retrieves the latest build on its own, so I didn't think I needed to pass anything else in. Let me know if you think I'm mistaken!

Reply:
Not sure if I understand why a new `isRestarting` is needed since you can do `restartWorkspace.isLoading` 🤔

Reply:
@BrunoQuaresma That would be true if we didn't have to watch for build logs and then kick off a call to `API.startWorkspace` as a side effect in the `onSuccess` callback. Unfortunately, `restartWorkspace.isLoading` isn't going to capture the time spend waiting for these requests.